SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -214,9 +215,11 @@ class UITable extends Component {
             onEnter={() => {
               const maxVisibleRows = this.state.chunkNum * this.props.rowChunkSize;
               if (maxVisibleRows < this.state.data.length) {
-                this.setState({
-                  chunkNum: this.state.chunkNum + 1
-                });
+                setTimeout(() => {
```

Comment:
You can use _.defer() here instead of a 0 timeout.
http://underscorejs.org/#defer